packages/pxweb2/src/app/components/FilterSidebar/FilterSidebar.module.scss

```diff
@@ -0,0 +1,28 @@
+@use '$ui/src/lib/text-styles.scss';
+
+.sideBar {
+  grid-area: sideBar;
+  background-color: var(--px-color-surface-subtle);
+  padding: 0 30px 0 20px;
+  overflow-y: auto;
+  position: sticky;
+  top: 0;
+  height: 100%;
+}
+
+.filterList {
+  margin: 20px 0 0;
+  padding: 0;
+  list-style-type: none;
+  & > li {
+    background-color: white;
+    margin: 10px;
+    padding: 5px;
+    border-color: black;
+    border-style: solid;
+    border-width: 1px;
+    border-radius: 7px;
```

Comment:
Make sure you use the specified border radiuses in the variables. If this is an exception, ignore my comment